SPECS/libmetalink/libmetalink.spec

```diff
@@ -0,0 +1,55 @@
+Summary:        libmetalink is a Metalink library written in C language. It is intended to provide the programs written in C to add Metalink functionality such as parsing Metalink XML files.
+Name:           libmetalink
```

Comment:
You'll probably also need an entry in `licenses.json` and the markdown file but the PR check will verify that and give you more details.